l10n_fr_payment_payfip/controllers/main.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import pprint
+import werkzeug
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.payment.models.payment_acquirer import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class PayFIPController(http.Controller):
+    _payment_url = '/payment/payfip/pay'
+    _return_url = '/payment/payfip/dpn'
+    _notification_url = '/payment/payfip/ipn'
+
+    @http.route(_payment_url, type='http', auth='none', methods=['GET', 'POST'], csrf=False)
```

Comment:
I don't rememeber the difference In v15 on comNpay module I'm using auth public here

https://github.com/Moka-Tourisme/payment/pull/2/files#diff-60f6b673b78075d3cfe7e7d4cc0adc2343187e1d49c7b54da9e3db7e425f69eaR17

```suggestion
    @http.route(_payment_url, type='http', auth='public', methods=['GET', 'POST'], csrf=False)
```

Reply:
in fact I don't remember when is call _payment_url ? it depends who is calling this endpoint, odoo or payfip?

`public` should be use when  call do not have session so at least on `_notification_url` end point should be enough :thinking: